lib/src/Uncertainty/Algorithm/Sensitivity/HSICEstimatorImplementation.cxx

```diff
@@ -256,6 +329,17 @@ Point HSICEstimatorImplementation::getPValuesPermutation() const
   return PValuesPermutation_;
 }
 
+/* Is it safe to compute the permutation p values in parallel? */
+Bool HSICEstimatorImplementation::isParallel() const
+{
+  bool isParallel_ = true;
+  for (UnsignedInteger i = 0; i < covarianceList_.getSize(); ++i)
+  {
+    isParallel_ = isParallel_ && covarianceList_[i].getImplementation()->isParallel();
+  }
+  return isParallel_;
```

Comment:
```suggestion
  Bool isParallel = true;
  for (UnsignedInteger i = 0; i < covarianceList_.getSize(); ++i)
  {
    isParallel = isParallel && covarianceList_[i].getImplementation()->isParallel();
  }
  return isParallel;
```
 - Bool needs to be written with an uppercase B (because "Bool" it the OT name for "bool", just like "Scalar" is the OT name for "float")
 - `isParallel` is no attribute, so no underscore is needed.